tensorflow_addons/optimizers/lookahead.py

```diff
@@ -16,11 +16,12 @@
 import tensorflow as tf
 from tensorflow_addons.utils import types
```

Comment:
isn't required to update types.Optimizer?

Reply:
good catch! done